docs/docs/monitoring/analytics/example-queries.mdx

```diff
@@ -0,0 +1,194 @@
+---
+id: example-queries
+sidebar_label: Example queries
+title: Example queries
+description:
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+
+Below you can find inspiration on how to analyze the conversations your assistant is having.
+Example queries using SQL are provided, together with example visualisation in Metabase.
+
+### Number of sessions per month
+
+It's common to look at high level usage metrics of your chatbot. A metric you could look at is number of
+sessions per month. Here is how it could look like:
+
+
+```sql
+SELECT
+  date_trunc('month', "public"."rasa_session"."timestamp") AS "first_seen",
+  count(*) AS "count"
+FROM "public"."rasa_session"
+GROUP BY 1
+ORDER BY 1 ASC
+```
+
+<div align="center">
+  <img
+    alt="Example result for a query showcasing the number of users per day."
+    src={useBaseUrl("/img/analytics/graph-number-sessions-month.png")}
+    width="100%"
+  />
+</div>
+
+### Number of sessions per channel
+
+If you're connecting your assistant to multiple channels, another metric that could be interesting to look at
+is the number of sessions per channel, let's say per week. You'd get the following query:
+
+```sql
+SELECT
+  "public"."rasa_user_message"."channel" AS "channel",
+  "public"."rasa_user_message"."timestamp" AS "timestamp",
+  count(distinct "public"."rasa_user_message"."session_id") AS "count"
+FROM "public"."rasa_user_message"
+GROUP BY 1, 2
+ORDER BY 1 ASC, 2 ASC
+```
+
+
+<div align="center">
+  <img
+    alt="Example result for a query showcasing the number of users per day."
+    src={useBaseUrl("/img/analytics/graph-number-sessions-channel.png")}
+    width="100%"
+  />
+</div>
+
+
+### Top N intents
+
+When improving your assistant, it's common to look at variety of intents your users express. The Top 5 intents metric
```

Comment:
"the variety"

Reply:
"The top 5 intent metrics" instead of "intents metric"

Reply:
d161380a4a5